librarymanagement/src/main/contraband/librarymanagement.json

```diff
@@ -267,23 +267,34 @@
-        { "name": "isTransitive",      "type": "boolean",                             "default": "true",                             "since": "0.0.1" },
-        { "name": "isForce",           "type": "boolean",                             "default": "false",                            "since": "0.0.1" },
-        { "name": "explicitArtifacts", "type": "sbt.librarymanagement.Artifact*",     "default": "Vector.empty",                     "since": "0.0.1" },
-        { "name": "inclusions",        "type": "sbt.librarymanagement.InclExclRule*", "default": "Vector.empty",                     "since": "0.0.1" },
-        { "name": "exclusions",        "type": "sbt.librarymanagement.InclExclRule*", "default": "Vector.empty",                     "since": "0.0.1" },
-        { "name": "extraAttributes",   "type": "Map[String, String]",                 "default": "Map.empty",                        "since": "0.0.1" },
-        { "name": "crossVersion",      "type": "sbt.librarymanagement.CrossVersion",  "default": "sbt.librarymanagement.Disabled()", "since": "0.0.1" },
-        { "name": "branchName",        "type": "Option[String]",                      "default": "None",                             "since": "0.0.1" }
+        { "name": "configurations",    "type": "Option[String]",                         "default": "None",                             "since": "0.0.1" },
+        { "name": "isChanging",        "type": "boolean",                                "default": "false",                            "since": "0.0.1" },
+        { "name": "isTransitive",      "type": "boolean",                                "default": "true",                             "since": "0.0.1" },
+        { "name": "isForce",           "type": "boolean",                                "default": "false",                            "since": "0.0.1" },
+        { "name": "explicitArtifacts", "type": "sbt.librarymanagement.Artifact*",        "default": "Vector.empty",                     "since": "0.0.1" },
+        { "name": "inclusions",        "type": "sbt.librarymanagement.InclExclRule*",    "default": "Vector.empty",                     "since": "0.0.1" },
+        { "name": "exclusions",        "type": "sbt.librarymanagement.InclExclRule*",    "default": "Vector.empty",                     "since": "0.0.1" },
+        { "name": "extraAttributes",   "type": "Map[String, String]",                    "default": "Map.empty",                        "since": "0.0.1" },
+        { "name": "crossVersion",      "type": "sbt.librarymanagement.CrossVersion",     "default": "sbt.librarymanagement.Disabled()", "since": "0.0.1" },
+        { "name": "branchName",        "type": "Option[String]",                         "default": "None",                             "since": "0.0.1" },
+        { "name": "checksum",          "type": "Option[sbt.librarymanagement.Checksum]", "default": "None",                             "since": "0.0.1" }
```

Comment:
I think this is at a wrong layer. First, a module may consist of many artifacts.
Second, it's an addressing mechanism, so you won't know the checksum up front.